check_ssl.sh

```diff
@@ -118,15 +120,15 @@ DATE_DIFFERENCE_DAYS=$((${DATE_DIFFERENCE_SECONDS}/60/60/24))
 # OUTPUT |
 #---------
 if [[ "${DATE_DIFFERENCE_DAYS}" -le "${CRITICAL_DAYS}" && "${DATE_DIFFERENCE_DAYS}" -ge "0" ]]; then
-	echo -e "CRITICAL: Cert will expire on: "${DATE_EXPIRE_FORMAT}""
+	echo -e "CRITICAL: Cert "$COMMON_NAME" will expire on: "${DATE_EXPIRE_FORMAT}""
```

Comment:
doesn't the curly brackets "{}" make the difference?
"${DATE_EXPIRE_FORMAT}" vs "$DATE_EXPIRE_FORMAT".

there were no intentions other than I had to put everything in quotation marks, thus ensuring that they are 100% the expected var.

Reply:
I put the quotation marks there to be somewhat consistent with your coding style (and did not notice the curly brackets). 
Would you prefer to have the final plugin output without the quotes or with quotes (`\"$COMMON_NAME\"`)?

By the way, I just noticed that the "subject" output of the openssl binary changed from version 1.0 to 1.1.
CentOS7 (**openssl 1.0.2k**)
`subject= /C=DE/O=Example Org/OU=IT/CN=vhost.example.com`

CentOS8 (**openssl 1.1.1g**)
`subject=C = DE, O = Example Org, OU = IT, CN = vhost.example.com`

Let me address that as well before accepting the pull request.

Reply:
> doesn't the curly brackets "{}" make the difference?
"${DATE_EXPIRE_FORMAT}" vs "$DATE_EXPIRE_FORMAT".

They don't. Curly braces are mainly for [parameter expansion](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html), or simply delimiting a variable name (e.g. `${foo}bar`). They don't otherwise prevent the value from being split, expanded or interpreted&mdash;for that, we still need proper quoting.

> Would you prefer to have the final plugin output without the quotes or with quotes (`\"$COMMON_NAME\"`)?

Given the original intention, I'd suggest not doing any quotes there; the final line should thus probably look something like this:

```sh
    echo -e "CRITICAL: Cert $COMMON_NAME will expire on: $DATE_EXPIRE_FORMAT"
```

That being said, this mixes your contribution with fixing a pre-existing issue, so perhaps splitting it up into two commits might be clean (@tongpu, any opinions?)

Reply:
Ok, I completely removed the quotes from my `$COMMON_NAME` insertion and leave it up to you to take care of the preexisting `${DATE_EXPIRE_FORMAT}`.

Reply:
Thanks a lot!

Reply:
@ayekat Regarding "fixing pre-existing issues": I wouldn't be too hard about that.

Reply:
Yeah, I think we can address those in a separate PR. It's not related to the features added here.